tools/unitctl/unitctl/src/cmd/instances.rs

```diff
@@ -22,66 +22,69 @@ pub(crate) async fn cmd(args: InstanceArgs) -> Result<(), UnitctlError> {
                 } else if !PathBuf::from(application).as_path().exists() {
                     eprintln!("application path must exist");
                     Err(UnitctlError::NoFilesImported)
-
                 } else {
                     let addr = ControlSocket::parse_address(socket);
                     if let Err(e) = addr {
-                        return Err(UnitctlError::UnitClientError{source: e});
+                        return Err(UnitctlError::UnitClientError { source: e });
                     }
 
                     // validate we arent processing an abstract socket
                     if let ControlSocket::UnixLocalAbstractSocket(_) = addr.as_ref().unwrap() {
-                        return Err(UnitctlError::ControlSocketError{
+                        return Err(UnitctlError::ControlSocketError {
                             kind: ControlSocketErrorKind::General,
                             message: "cannot pass abstract socket to docker container".to_string(),
-                        })
+                        });
                     }
 
                     // warn user of OSX docker limitations
                     if let ControlSocket::UnixLocalSocket(ref sock_path) = addr.as_ref().unwrap() {
                         if cfg!(target_os = "macos") {
-                            return Err(UnitctlError::ControlSocketError{
+                            return Err(UnitctlError::ControlSocketError {
                                 kind: ControlSocketErrorKind::General,
-                                message: format!("Docker on OSX will break unix sockets mounted {} {}",
-                                                 "in containers, see the following link for more information",
-                                                 "https://github.com/docker/for-mac/issues/483"),
-                            })
+                                message: format!(
+                                    "Docker on OSX will break unix sockets mounted {} {}",
```

Comment:
```suggestion
                                    "Docker on macOS will break unix sockets mounted {} {}",
```

(Sorry)

Reply:
"unix sockets" should probably be "UNIX domain sockets"